lib/cuda/cudaCorrelator.cpp

```diff
@@ -69,7 +67,7 @@ cudaCorrelator::cudaCorrelator(Config& config, const std::string& unique_name,
                                               n2k_dimnames, *this);
     }()),
     n2correlator(_num_elements, _num_local_freq) {
-    if (_samples_per_data_set % _sub_integration_ntime)
+    if (_num_times % _sub_integration_ntime)
```

Comment:
_num_times could probably also use a better name, or docstring (number of times of what, and does this coincide with other num_times elsewhere)?